feature/gribi/ate_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -324,20 +349,20 @@ func createFlow(t *testing.T, ate *ondatra.ATEDevice, top *ondatra.ATETopology,
 }
 
 // validateTrafficFlows verifies that the flow on ATE and check interface counters on DUT
-func validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, flow *ondatra.Flow, drop bool, dPort []string) {
+func validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, flow *ondatra.Flow, dPort []string) {
 	ate.Traffic().Start(t, flow)
 	time.Sleep(60 * time.Second)
 	ate.Traffic().Stop(t)
 	flowPath := gnmi.OC().Flow(flow.Name())
-	got := gnmi.Get(t, ate, flowPath.LossPct().State())
-	if drop {
-		if got != 100 {
-			t.Fatalf("Traffic passing for flow %s got %f, want 100 percent loss", flow.Name(), got)
-		}
-	} else {
-		if got > 0 {
-			t.Fatalf("LossPct for flow %s got %f, want 0", flow.Name(), got)
-		}
+	val, _ := gnmi.Watch(t, ate, flowPath.LossPct().State(), 5*time.Minute, func(val *ygnmi.Value[float32]) bool {
```

Comment:
Why wait for 5 minutes after the traffic has been stopped?

Reply:
also, the watch here is for traffic generator and we want to ensure test does not fail in case ATE is slow.

Reply:
I'm just surprised by the fact that ATE could take up to 5 minutes to generate packet loss statistics? Is that the case?

Reply:
In most test cases Get seems to be sufficient for this purpose, in a couple cases i see a 1-minute timer.. so If there's no strict need for 5 minute i'd prefer to keep tests consistent and use up to 1 minute or Get

Reply:
we are trying deal with flakyness (not a normal flow), otherwise you are right and get work for most of the time, but it also fails intermittently. if the ATE is fast, then the await call with return soon and will not wait for 5 minutes (this is the a safe upper bound). Concerning the consistency we are going to convert all ATE call to use await to deal with flakiness.

Reply:
sgtm.

Reply:
decreased to two minutes, since i noticed when there is a traffic loss, the call wait for 5 minutes because of the check for predicate.